README.md

```diff
@@ -35,18 +25,26 @@ module "codepipeline" {
 ## Inputs
 | Name | Type |Description | Default |
 | --- | --- | --- | --- |
-| app_name | string | The name of the application. |
-| account_env | string | The environment (dev/prd) of the account you're deploying to (should match the ACS module). |
-| env_tag | string | The environment tag for the resources. |
-| data_sensitivity_tag | string | The data-sensitivity tag for the resources. | confidential |
-| repo_name | string | The name of the repository of the project. |
-| branch | string | The name of the branch you want to trigger the pipeline. |
-| deploy_provider | string | The provider for the deploy stage of the pipeline. If set to null this will skip setting up a deploy phase. | null |
-| deploy_configuration | any | The configuration for the deploy provider. If set to null this will skip setting up a deploy phase. | null |
-| build_buildspec | string | The file to use for the build phase. | buildspec.yml |
-| role_permissions_boundary_arn | string | The role permissions boundary ARN. |
-| github_token | string | The GitHub token associated with the AWS account. |
-| power_builder_role_arn | string | The ARN for the PowerBuilder role. |
+| pipeline_name | string | Unique name for the pipeline. No spaces. | |
+| acs_env | string | Environment of the AWS Account (e.g. dev, prd) | |
```

Comment:
Would you remind removing `acs_env` this and the variable. It isn't used now.

Reply:
removed